src/types.ts

```diff
@@ -15,14 +15,26 @@ export const chainExplorerSchema = z.object({
   browserUrl: z.string().url(),
 });
 
+export const chainProviderSchema = z
+  .object({
+    alias: z.string(),
+    homepageUrl: z.string().url().optional(),
+    rpcUrl: z.string().url().optional(),
+  })
+  .refine(
```

Comment:
Note that either `homepageUrl` or `rpcUrl` is required. Having providers that only have an `alias` field feels like a mistake

Reply:
Agreed